src/NServiceBus.Core.Tests/Pipeline/Outgoing/OutgoingReplyContextTests.cs

```diff
@@ -7,30 +7,13 @@
     public class OutgoingReplyContextTests
     {
         [Test]
-        public void ShouldShallowCloneHeaders()
-        {
-            var message = new OutgoingLogicalMessage(typeof(object), new object());
-            var options = new ReplyOptions();
-            options.SetHeader("someHeader", "someValue");
-
-            var testee = new OutgoingReplyContext(message, options, new RootContext(null, null, null));
-            testee.Headers["someHeader"] = "updatedValue";
-            testee.Headers["anotherHeader"] = "anotherValue";
-
-            Assert.AreEqual("someValue", options.OutgoingHeaders["someHeader"]);
-            Assert.IsFalse(options.OutgoingHeaders.ContainsKey("anotherHeader"));
-            Assert.AreEqual("updatedValue", testee.Headers["someHeader"]);
-            Assert.AreEqual("anotherValue", testee.Headers["anotherHeader"]);
-        }
-
-        [Test]
         public void ShouldShallowCloneContext()
         {
             var message = new OutgoingLogicalMessage(typeof(object), new object());
             var options = new ReplyOptions();
             options.Context.Set("someKey", "someValue");
 
-            var testee = new OutgoingReplyContext(message, options, new RootContext(null, null, null));
+            var testee = new OutgoingReplyContext(message, "id", options.OutgoingHeaders, options.Context, new RootContext(null, null, null));
```

Comment:
Nitpick: `message-id` or similar would be a better value to avoid the reader guessing